develop/toolkit/api-libraries/smart-contracts/ethers-js.md

```diff
@@ -0,0 +1,146 @@
+
+Here's a sample Solidity contract (`Storage.sol`) to be compiled and deployed to Asset Hub. This contract's functionality stores a number and permits users to update it with a new value.
+
+```solidity
+--8<-- 'code/develop/smart-contracts/evm-toolkit/ethers-js/Storage.sol'
+```
+
+To compile this contract, use the following script:
+
+```js
+--8<-- 'code/develop/smart-contracts/evm-toolkit/ethers-js/compile.js'
+```
+
+Note that the script above is tailored to the `Storage.sol` contract. It can be adjusted for other contracts by changing the file name or modifying the ABI and bytecode paths accordingly.
+
+After executing the script, the Solidity contract will be compiled into the required `polkavm` bytecode format. The ABI and bytecode will be saved into files with `.json` and `.polkavm` extensions, respectively. You can now proceed with deploying the contract to Asset Hub network, as outlined in the next section.
+
+## Contracts Deployment
+
+To deploy your compiled contract to Asset Hub, you'll need a wallet with a private key to sign the deployment transaction.
+
+The deployment script can be broken down into key components:
+
+1. First, set up the required imports and utilities:
+
+    ```js
+    --8<-- 'code/develop/smart-contracts/evm-toolkit/ethers-js/deploy.js:1:5'
+    ```
+
+2. Create a provider to connect to Asset Hub network:
+
+    ```js
+    --8<-- 'code/develop/smart-contracts/evm-toolkit/ethers-js/deploy.js:6:14'
+    ```
+ 
+3. Set up functions to read contract artifacts:
+
+    ```js
+    --8<-- 'code/develop/smart-contracts/evm-toolkit/ethers-js/deploy.js:16:42'
+    ```
+
+4. Create the main deployment function:
+
+    ```js
+    --8<-- 'code/develop/smart-contracts/evm-toolkit/ethers-js/deploy.js:44:79'
+    ```
+
+5. Configure and execute the deployment:
+
+    ```js
+    --8<-- 'code/develop/smart-contracts/evm-toolkit/ethers-js/deploy.js:81:89'
+    ```
+
+    !!! note
+        Ensure to replace the `INSERT_MNEMONIC` placeholder with your actual mnemonic.
+
+Here's the complete deployment script combining all the components above:
+
+??? code "deploy.js"
+
+    ```js
+    --8<-- 'code/develop/smart-contracts/evm-toolkit/ethers-js/deploy.js'
+    ```
+
+After running this script, your contract will be deployed to Asset Hub, and its address will be saved in `contract-address.json` within your project directory. You can use this address for future contract interactions.
+
+## Interact with the Contract
+
+Once the contract is deployed, you can interact with it by calling its functions. For example, to set a number, read it and then modify that number by its double, you can use the following script:
+
+```js
+--8<-- 'code/develop/smart-contracts/evm-toolkit/ethers-js/checkStorage.js'
+```
+
+Ensure you replace the `INSERT_MNEMONIC`, `INSERT_CONTRACT_ADDRESS` and `INSERT_ADDRESS_TO_CHECK` placeholders with actual values. Also, the contract ABI file (`Storage.json`) should be correctly referenced.
+
+## Conclusion
```

Comment:
Let's trade this out for a ## Where to Go Next for consistency with our other pages. See example page: /develop/toolkit/api-libraries/py-substrate-interface/#where-to-go-next